tests/contest/contest/src/tests/process/process_test.rs

```diff
@@ -0,0 +1,47 @@
+use crate::utils::test_inside_container;
+use anyhow::{bail, Context, Ok, Result};
+use oci_spec::runtime::{ProcessBuilder, Spec, SpecBuilder};
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+use test_framework::{test_result, Test, TestGroup, TestResult};
+
+fn create_spec() -> Result<Spec> {
+    let spec = SpecBuilder::default()
+        .process(
+            ProcessBuilder::default()
+                .cwd("/test")
+                .env(vec!["testa=valuea".into(), "testb=123".into()])
+                .build()
+                .expect("error in creating process config"),
+        )
+        .build()
+        .context("failed to build spec")?;
+
+    Ok(spec)
+}
+
+fn process_test() -> TestResult {
+    let spec = test_result!(create_spec());
+    test_inside_container(spec, &|_| {
+        match fs::create_dir("/test") {
```

Comment:
This should not be `/test`, you should take the bundle param in the closure and append `/test` and create that dir. This callback is run in host's fs before running the container, so you should be using bundle path and create this dir.

Reply:
@YJDoc2  
Sorry, I dont understand test freamwork well yet.
Can you please advise where I should create the directory?

Reply:
Hey @sat0ken , not sure if you're still working on this or done, but-
The approach you have added in https://github.com/youki-dev/youki/pull/2968/commits/988b9330955e0e1caf88befdd2fdcd9c0ec24111 is still wrong. You are getting the cwd from spec, which will still be inside the container root. What you need is basically -
```rust
 test_inside_container(spec, &|bundle| {
fs::create_dir(bundle.join("test"))
```
take `bundle` as the closure param, join test to it and then create that dir.

Reply:
I fixed this.
https://github.com/youki-dev/youki/pull/2968/commits/67fdf520ec714a8e3dab12f964d2bdff6a0e910d